### DISCUSSION TITLE
Welcome to AutorizePro Discussions! 👏

### DISCUSSION THREAD

**Discussion by sule01u** [sule01u/AutorizePro#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Quitten** [comment]
@sule01u - I create [this issue](https://github.com/WuliRuler/AutorizePro/issues/41), would be happy to get a PR for merging our projects together, there is no reason to keep your version not up to date with mine, let me know WDYT and lets get in touch, sent you email as well
